from helper import unique_error_loggers

errors = unique_error_loggers()
for e in errors:
    print(e)